LSE-61.tex

```diff
@@ -3642,6 +3642,786 @@ \subsubsection{No Limit on Data Access Centers}
+
+\begin{parameters}
+Minimum number of simultaneous users querying the Prompt Products Database.
+&
+20
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+l1QueryUsers
+} \\\hline
+Maximum time allowed for retrieving results of a query of the Prompt Products Database.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+l1QueryTime
+} \\\hline
+\end{parameters}
+
+\subsubsection{DM EFD Query Performance}
+
+\label{DMS-REQ-0358}
+\textbf{ID:} DMS-REQ-0358 (Priority: 1a)
+
+\textbf{Specification:}
+The DM copy of the EFD shall support at least \textbf{dmEfdQueryUsers} simultaneous queries, assuming each query lasts no more than \textbf{dmEfdQueryTime}.
+
+\begin{parameters}
+Maximum time allowed for retrieving results of a DM EFD query.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+dmEfdQueryTime
+} \\\hline
+Minimum number of simultaneous users querying the DM EFD.
+&
+5
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+dmEfdQueryUsers
+} \\\hline
+\end{parameters}
+
+\subsection{Image Access Performance Metrics}
+
+\subsubsection{Retrieval of postage stamp light curve images}
+
+\label{DMS-REQ-0375}
+\textbf{ID:} DMS-REQ-0375 (Priority: 2)
+
+\textbf{Specification:}
+Postage stamp cutouts, sized to match those in alert packets, of all observations of a single Object shall be retrievable within \textbf{postageStampRetrievalTime}, with \textbf{postageStampRetrievalUsers} simultaneous requests.
+
+\textbf{Discussion:}
+These postage stamps can be obtained from lossy compressed PVIs.
+
+\begin{parameters}
+Maximum time allowed for retrieving a set of postage stamp images of a single Object.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+postageStampRetrievalTime
+} \\\hline
+Minimum number of simultaneous users retrieving a set of postage stamp images.
+&
+10
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+postageStampRetrievalUsers
+} \\\hline
+\end{parameters}
+
+\subsubsection{Retrieval of CCD-sized images}
+
+\label{DMS-REQ-0374}
+\textbf{ID:} DMS-REQ-0374 (Priority: 1b)
+
+\textbf{Specification:}
+A Processed Visit Image of a single CCD shall be retrievable using the VO SIAv2 protocol within \textbf{pviRetrievalTime} with \textbf{pviRetrievalUsers} simultaneous requests.
+
+\textbf{Discussion:}
+\begin{parameters}
+Minimum number of simultaneous users retrieving a single PVI image.
+&
+20
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+pviRetrievalUsers
+} \\\hline
+Maximum time allowed for retrieving a PVI image of a single CCD from a single visit
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+pviRetrievalTime
+} \\\hline
+\end{parameters}
+
+\subsubsection{Retrieval of focal-plane visit images}
+
+\label{DMS-REQ-0376}
+\textbf{ID:} DMS-REQ-0376 (Priority: 1b)
+
+\textbf{Specification:}
+All Processed Visit Images for a single visit, including mask and variance planes, shall be retrievable using the VO SIAv2 protocol within \textbf{allPviRetrievalTime} with \textbf{allPviRetrievalUsers} simultaneous requests.
```

Comment:
"All Processed Visit Images for a single visit that are available in cache, including variance and mask planes, shall be identifiable with a single IVOA SIAv2 service query and retrievable, using the link(s) provided in the response, within \textbf{allPviRetrievalTime}.  This requirement shall be met for up to \textbf{allPviRetrievalUsers} simultaneous requests for distinct focal-plane PVI sets."

Reply:
3.3 Gpix * 12 bytes/pixel * 20 users / 60 seconds is about 13 GB/s of external bandwidth to the public Internet required to satisfy this requirement, before any compression.  This is in parallel with the (much smaller) load from the single-CCD requirement, the potentially large load from database result sets, and whatever the bulk download service requires.  Let's be sure we've accounted for all these.

Reply:
@leannep was wondering if we should also have a requirement that explicitly mentions that we have to also do tests of some of these other requirements in parallel. Obviously if we can't meet the standalone requirement we'll fail the simultaneous requirement.